src/os_utils.py

```diff
@@ -1,19 +1,30 @@
 import os
+from functools import cache
 
+from consts import BLOCKED_LABELS
 
+
+@cache
 def get_version() -> str:
     """
     Wrapper for CLUSTER_SECRET_VERSION variable environment
     """
     return os.getenv('CLUSTER_SECRET_VERSION', '0')
 
 
+@cache
 def get_replace_existing() -> bool:
-
     replace_existing = os.getenv('REPLACE_EXISTING', 'false')
     return replace_existing.lower() == 'true'
 
 
+@cache
+def get_blocked_labels() -> list[str]:
+    blocked_labels = os.getenv('BLOCKED_LABELS', ','.join(BLOCKED_LABELS))
+    return [label.strip() for label in blocked_labels.split(',')]
```

Comment:
two things I noticed:

- joining and then splitting again for the default case 
- the return type with this implementation would be a generator

Reply:
would this be more Pythonic to you?

```python
os.getenv('BLOCKED_LABELS', '').split(',') or BLOCKED_LABELS
```


`return (...)` would be a generator, `[...]` produces a list. I'll run mypy on it though.

Reply:
> `return (...)` would be a generator, `[...]` produces a list. I'll run mypy on it though.

indeed, wow learned something

Reply:
the walrus :) noice.